kuma/javascript/src/payments/__tests__/thank-you.test.jsx

```diff
@@ -0,0 +1,13 @@
+//@flow
+import React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow';
+import ThankYouPage from '../thank-you.jsx';
+
+describe('Payments Thank You page', () => {
+    test('it renders', () => {
+        const renderer = new ShallowRenderer();
```

Comment:
I originally used it because I care mainly about whether or not the high-level items rendered, like `<FeedbackForm />`. 

I don't find snapshot testing very useful, so getting rid of the `__tests__` folder (since snapshots don't have a place to hide) encouraged me to re-write this test so it doesn't rely on snapshots.